tests/ttnn/unit_tests/operations/test_reduction.py

```diff
@@ -48,3 +48,25 @@ def test_var(device, batch_size, h, w, dim):
 
     output_tensor = ttnn.to_torch(output_tensor)
     assert_with_pcc(torch_output_tensor, output_tensor, pcc=0.99)
+
+
+@pytest.mark.parametrize("batch_size", [1, 16, 1, 16])
+@pytest.mark.parametrize("c", [1, 8, 1, 16])
+@pytest.mark.parametrize("h", [32, 64, 41, 37])
+@pytest.mark.parametrize("w", [32, 64, 31, 63])
+@pytest.mark.parametrize("dim", [[0, 1, 2, 3]])
```

Comment:
dim should go through None, [], and [0,1,2,3]

Reply:
Given the issue with [] please just leave as is.

Reply:
adding only None for now. Will address [] in next PR